include/deal.II/multigrid/mg_constrained_dofs.h

```diff
@@ -196,6 +205,12 @@ class MGConstrainedDoFs : public Subscriptor
   const AffineConstraints<double> &
   get_level_constraint_matrix(const unsigned int level) const;
 
+  /**
+   * Return the user defined constraint matrix for a given level.
+   */
```

Comment:
Mention that it is filled by set_user_constraints() and that is does not contain dirichlet values. or does it?